src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -42,7 +42,7 @@ public static class UriHelper
         /// <param name="query"></param>
         /// <param name="fragment"></param>
         /// <returns></returns>
-        public static string Encode(
+        public static string BuildAbsolute(
             string scheme,
             HostString host,
```

Comment:
:ok:, less concerned about this guy. Can always add the additional overloads later if it turns out to be a burden on users.